health/healthcheck_test.go

```diff
@@ -21,12 +21,14 @@ func (mc mockConsumer) ConnectivityCheck() error {
 	return mc.err
 }
 
-func (mc mockConsumer) StartListening(messageHandler func(message kafka.FTMessage) error) {
-	return
+func (mc mockConsumer) MonitorCheck() error {
+	return mc.err
+}
+
+func (mc mockConsumer) Start(messageHandler func(message kafka.FTMessage)) {
```

Comment:
With the introduction of the local interfaces all other functions i.e. `Start(), `SendMessage()`, Close()` etc. can be removed.